funwithphysics/src/Components/PhysicsTopic/index.js

```diff
@@ -42,35 +42,35 @@ function PhysicsTopic() {
 				<div className="container">
 					<div className="row">
 					<PhysicsTopicItem name="Classical mechanics" link="/classicalmechanics" icon="icon-shop" image={classicalMechanics} >
-							<p>Classical mechanics is a theory describing the motion of macroscopic objects, projectile motions of parts of machinery and astronomical objects and also rotatory motion of machines. </p>
+							<p>Classical mechanics is a theory describing the motion of macroscopic objects, projectile motions of parts of machinery and astronomical objects and also rotatory motion of machines. For more Information visit <a href="https://en.wikipedia.org/wiki/Classical_mechanics" target="_blank" rel="noreferrer">Wikipedia</a></p>
 							<p id="credit">Photo by <a href="https://unsplash.com/@gabriele_macera?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Gab</a> on <a href="https://unsplash.com/s/photos/engine?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></p>
 							</PhysicsTopicItem>
 					<PhysicsTopicItem name="Thermodynamics" link="/thermodynamics" icon="icon-heart4" image={thermodynamics}>
-							<p>Thermodynamics is the branch of physics that deals with the transfer of heat and energy from one body to another.</p>
+							<p>Thermodynamics is the branch of physics that deals with the transfer of heat and energy from one body to another.  For more Information visit <a href="https://en.wikipedia.org/wiki/Thermodynamics" target="_blank" rel="noreferrer">Wikipedia</a></p>
 							 <p id="credit">  Photo by <a href="https://unsplash.com/@vivianeokubo?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Viviane Okubo</a> on <a href="https://unsplash.com/s/photos/heat-transfer?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText">Unsplash</a></p>
   							</PhysicsTopicItem>
 						<PhysicsTopicItem name="Relativity" link="/relativity" icon="icon-lab2" image={relativity}>
-							<p>Relativity is the notion that the laws of physics are the same everywhere, be it on Earth or in a far off corner of the universe.</p>
+							<p>Relativity is the notion that the laws of physics are the same everywhere, be it on Earth or in a far off corner of the universe. For more Information visit <a href="https://en.wikipedia.org/wiki/Theory_of_relativity" target="_blank" rel="noreferrer">Wikipedia</a></p>
 						    <p id="credit">Image by <a href="https://pixabay.com/users/stux-12364/?utm_source=link-attribution&amp;utm_medium=referral&amp;utm_campaign=image&amp;utm_content=4129246">Thanks for your Like • donations welcome</a> from <a href="https://pixabay.com/?utm_source=link-attribution&amp;utm_medium=referral&amp;utm_campaign=image&amp;utm_content=4129246">Pixabay</a></p>
```

Comment:
Remove the Wikipedia link from this page.

Reply:
ok

Reply:
done